gateway-package/src/main/resources/bin/start.sh

```diff
@@ -100,6 +100,7 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${GATEWAY_CODE} java \
     -Dapiml.security.ssl.nonStrictVerifySslCertificatesOfServices=${NONSTRICT_VERIFY_CERTIFICATES:-false} \
     -Dapiml.security.auth.zosmf.serviceId=${APIML_ZOSMF_ID:-zosmf} \
     -Dapiml.security.auth.provider=${APIML_SECURITY_AUTH_PROVIDER} \
+    -Dapiml.security.auth.jwtKeyAlias=${PKCS11_TOKEN_LABEL:-jwtsecret} \
```

Comment:
Will it still work if the user sets APIML_SECURITY_AUTH_JWT_KEY_ALIAS?

Reply:
it will not work, this value will be replaced by the one in start script

Reply:
I can add this condition to still use if APIML_SECURITY_AUTH_JWTKEYALIAS if PKCS11_TOKEN_LABEL is missing 

if [[ -z ${PKCS11_TOKEN_LABEL} && ! -z ${APIML_SECURITY_AUTH_JWTKEYALIAS}]]
then
    PKCS11_TOKEN_LABEL=${APIML_SECURITY_AUTH_JWTKEYALIAS}
fi

Reply:
Thanks